open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/synchronization/model/AnalyticsMetadata.java

```diff
@@ -7,11 +7,13 @@
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Objects;
 
 import org.odpi.openmetadata.accessservices.analyticsmodeling.synchronization.IdMap;
 import org.odpi.openmetadata.accessservices.analyticsmodeling.synchronization.beans.SchemaAttribute;
 import org.odpi.openmetadata.accessservices.analyticsmodeling.utils.Constants;
 
+
 /**
  *	Base class for analytics metadata provides common attributes. 
  */
```

Comment:
Needs Jackson annotations

Reply:
I've added annotations but the class and sub classes are not exposed through REST api.

Reply:
Why have sub-classes if they are not consumable by callers?

Reply:
Sub classes are used explicitly not through base class. They are part of AnalyticsAsset and are aggregated as List<MetadataContainer> and List<MetadataItem> but not as List<AnalyticsMetadata> 
Sub-classes reuse common attributes and implement only specifics to reduce code duplication.
From my understanding, sub type annotation is mandatory if  List<AnalyticsMetadata> contains different sub types.